modeltranslation/fields.py

```diff
@@ -400,6 +447,27 @@ def __get__(self, instance, owner):
         return None
 
 
+class TranslatedManyToManyDescriptor(object):
+    """
+    A descriptor used to return correct related manager without language fallbacks.
+    """
+
+    def __init__(self, field_name, fallback_languages):
```

Comment:
```suggestion
    def __init__(self, field_name: str, fallback_languages: Iterable[str]):
```

Please, add types to as much new code as possible. It helps understand what's going on.

Reply:
Yes that's fair, I'll add types as much as possible.